server/app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.static(__dirname + '/../public'));
+// app.use(express.static(__dirname + '/../client/dist'));
+
+app.get('/', (req, res, err) => {
+    res.send(data);
+});
+
```

Comment:
This Code above doesn't do anything special.  

If your intention is to send the public directory with all it's content, the code `app.use(express.static(__dirname + '/../public'));` is doing just that.  However, your path should be that you specify here isn't configured correctly.

When using the above syntax, and appending the path `'/../public'` to the `__dirname'  you are effectively extending the path to the `sever.js` file to `server.js/../public`.  which is not what you want.  You need to use the path module provided by Node.

Reply:
Ahh, you changed it.  Great!!!